handbook/engineering/leadership/index.md

```diff
@@ -43,33 +43,33 @@ We have a 1hr sync every Monday at 9:30am PT.
 - This meeting is recorded and automatically posted to #eng-manager-team.
 - Our [meeting notes](https://docs.google.com/document/d/1nZTGua3cdMTJLwANlBZNX2uXiDXLuYDBkH9tehwdqTY/edit#) are also internally available to the whole team.
 
-### Weekly updates
+### Weekly highlight
 
-The goal of these weekly updates is to communicate each team's progress and tie that progress to larger company goals.
+Engineering managers send through a weekly highlight to [engineering-team-status@sourcegraph.com](https://groups.google.com/a/sourcegraph.com/g/engineering-team-status) every Friday by 1PM [PT](https://time.is/PT), or at EOD Thursday if you don't work on Friday. Delegate to someone else on your team to send the update when you are on vacation.
```

Comment:
```suggestion
Engineering managers send a weekly highlight to [engineering-team-status@sourcegraph.com](https://groups.google.com/a/sourcegraph.com/g/engineering-team-status) every Friday by 1PM [PT](https://time.is/PT), or at EOD Thursday if you don't work on Friday. Delegate to someone else on your team to send the update when you are on vacation.
```

Reply:
"Send through" sounds weird to me, but it might just be that it only sounds weird in American english? I don't know. 😄 Feel free to ignore.

Reply:
Is it weird to post the actual email address here? (If that google group rejects emails from non @sourcegraph.com accounts, then I'm not worried about it.)

Reply:
Email address doesn't accept outside emails.